src/main/java/com/eva/logic/commands/AddApplicationCommand.java

```diff
@@ -44,6 +51,10 @@ public AddApplicationCommand(Index index, Application application) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+        PanelState panelState = model.getPanelState();
+        if (!panelState.equals(APPLICANT_PROFILE) && !panelState.equals(APPLICANT_LIST)) {
+            throw new CommandException(MESSAGE_WRONG_PANEL);
```

Comment:
We should have a standardized error message when having a wrong panel related error (refer to viewCommand).
The error message can be formatted as such:
String.format(Messages.MESSAGE_INVALID_COMMAND_AT_PANEL, MESSAGE_WRONG_PANEL))